src/content/docs/contributing/modules.md

```diff
@@ -577,8 +563,8 @@ nf-test test --tag "<tool>/<sub-tool>" --profile docker
 
 ```yaml
 <tool>/<sub-tool>:
-  - modules/nf-core/<tool>/<sub-tool>/**
-  - modules/nf-core/<tool>/<sub-tool>/**
+  - modules/nf-core/<tool>/<sub-tool-1>/**
+  - modules/nf-core/<tool>/<sub-tool-2>/**
```

Comment:
What are these for?

Reply:
just a distinction for diff tools in a suite